main.go

```diff
@@ -84,4 +84,37 @@ func main() {
 		handleError(err)
 	}
 	fmt.Println("Number of transactions:", len(transactionsResp.Transactions))
+
+	// POST /payment_initiation/recipient/create
```

Comment:
We might want to gate the payment calls in this file behind a command line flag, since clients are not currently enabled for payment initiation in the sandbox environment by default.

Reply:
Will plan to tackle this improvement in a follow up PR in the interest of merging.